tests/python/test_linalg.py

```diff
@@ -304,10 +301,18 @@ def fill():
             assert m4[0][j, i] == int(i + 3 * j + 1)
 
 
+@test_utils.test()
+def test_init_matrix_from_vectors():
+    _test_init_matrix_from_vectors()
+
+
+@test_utils.test(real_matrix=True, real_matrix_scalarize=True)
+def test_init_matrix_from_vectors_matrix_scalarize():
+    _test_init_matrix_from_vectors()
+
+
 # TODO: Remove this once the APIs are obsolete.
-@pytest.mark.filterwarnings('ignore')
-@test_utils.test(arch=get_host_arch_list())
-def test_init_matrix_from_vectors_deprecated():
+def _test_init_matrix_from_vectors_deprecated():
```

Comment:
It seems that this test is exactly the same as the last one. Shall we only keep one of them?